agent/lib/kontena/workers/fluentd_worker.rb

```diff
@@ -0,0 +1,60 @@
+require 'fluent-logger'
+
+module Kontena::Workers
+  class FluentdWorker
+    include Celluloid
+    include Celluloid::Notifications
+    include Kontena::Logging
+
+    attr_reader :fluentd, :node_name, :queue
+
+    ##
+    # @param [Queue] queue
+    # @param [Boolean] autostart
+    def initialize(autostart = true)
+      @fluentd = nil
+      @queue = []
+      @forwarding = false
+      info 'initialized'
+      subscribe('agent:node_info', :on_node_info)
+      subscribe(Kontena::Workers::ContainerLogWorker::EVENT_NAME, :on_log_event)
```

Comment:
IMO it would be clearer to just use the literal `subscribe('container:log')` event name both here and in the publish. Using this kind of constant is harder to grep for.